pandas/core/arrays/categorical.py

```diff
@@ -2237,21 +2237,20 @@ def unique(self):
         order of appearance.
 
         >>> pd.Categorical(list('baabc'))
-        [b, a, c]
-        Categories (3, object): [b, a, c]
+        [b, a, a, b, c]
+        Categories (3, object): [a, b, c]
```

Comment:
I think there are missing `unique()` calls here (and then the output should be correct), since we are in the unique docstring

Reply:
Good catch!

I got into a tunnel vision of just correcting the output, I guess I didn't even looked at what is the docstring I'm editing